diff --git a/dist/index.mjs b/dist/index.mjs
index eab15c2373f44f0333fc4995e46588b25058f90d..94421f571efae37f23c544f69c236edf4e26f6f4 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -22,7 +22,7 @@ var isDebugMode = () => {
   const values = process.env.DEBUG?.toLocaleLowerCase().split(",") ?? [];
   return ["rsbuild", "builder", "*"].some((key) => values.includes(key));
 };
-var isDevDebugMode = () => process.env.DEBUG === "rspress-dev";
+var isDevDebugMode = () => (process.env.DEBUG) === "rspress-dev";
 var cleanUrl = (url) => url.replace(HASH_REGEXP, "").replace(QUERY_REGEXP, "");
 function slash(str) {
   return str.replace(/\\/g, "/");
